src/test/java/org/apache/commons/collections4/bloomfilter/SimpleBloomFilterTest.java

```diff
@@ -41,4 +41,16 @@ public void testMergeShortBitMapProducer() {
         assertTrue(filter.merge(producer));
         assertEquals(1, filter.cardinality());
     }
+
+    @Test
+    public void testCardinalityAndIsEmpty() {
```

Comment:
This is just doing what the parent class does. The extra test should be in its own fixture, e.g.
```java
@Test
public void testEmptyAfterMergeWithNothing() {
    // test the case where is empty after merge
    // in this case the internal cardinality == -1
    BloomFilter bf = createEmptyFilter(getTestShape());
    bf.merge(IndexProducer.fromIndexArray());
    assertTrue(bf.isEmpty());
}
```